### PR TITLE
sprite.svg file fix

### PR DESCRIPTION
**Pull request by harshil1793**
## Chromatic

https://1977-icon-file-fix--60f9b557105290003b387cd5.chromatic.com

---

## Description
This PR updates the Stencil configuration to ensure that the `sprite.svg` is referenced with the correct path.
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1977

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [ ] va-icon works locally, for Chromatic, in Storybook prod, and vets-website.

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by harshil1793** [comment]
For the local and chromatic it works fine. This attempt is to fix production bug.

File: packages/web-components/stencil.config.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const config: Config = {
     },
     {
       type: 'www',
+      dir: process.env.ENV === 'production' ? '/storybook' : '',
```
Comment (micahchiang) [inline packages/web-components/stencil.config.ts:55]: probably a silly question but where are we reading our env variables from? I looked around and couldn't determine where we may have `ENV` set. I didn't see it in any github action.

**Review by micahchiang — APPROVED** [review]
Let's fire this up and see if it resolves our issues on prod :rocket:

**Review by powellkerry — APPROVED** [review]

File: packages/web-components/stencil.config.ts (hunk 1 above)
Comment (harshil1793) [reply packages/web-components/stencil.config.ts:55]: @micahchiang Good point. While the PR is being reviewed, I've been trying to find an answer regarding adding environment variables.

**Review by micahchiang — APPROVED** [review]

**Pull request merged**